### DISCUSSION TITLE
GloVe or what?

### DISCUSSION THREAD

**Discussion by jenniewilliams** [explosion/spaCy#7780] [Help: Coding & Implementations] [answered]
When using en_core_web_lg (-2.3.1) are the resulting   .vector[:300] static vectors from GloVe or what?

**Comment by polm** [comment, marked as answer]
Yes, they use the GloVe Common Crawl vectors.

https://github.com/explosion/spacy-models/releases//tag/en_core_web_lg-2.3.1

**Reply by jenniewilliams** [reply]
Do you happen to now what, if any, context window was used when the embeddings were trained? I am training my own with a context window of 10, and comparing them, so wanted to be sure of parameter settings :)

**Reply by polm** [reply]
I do not. I recommend you check the source and paper linked from the GloVe homepage. 

https://nlp.stanford.edu/projects/glove/
